<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QFrame" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QGraphicsView" name="left_up">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="center_up">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="right_up">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="left">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="center">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="right">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="left_down">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="center_down">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="right_down">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="l_u_u">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_u_u">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="r_u_u">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="l_u_l">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_l_l">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="l_d_l">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="l_d_d">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_d_d">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="r_d_d">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="l_u_l_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="c_r_r">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="r_d_r">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>160</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spinLeft">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spinDown">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spinRight">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>330</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spinUp">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_of_edge">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
